tools/topology2/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;"
+set(TPLGS
+	"sof-cnl-nocodec\;sof-cnl-nocodec\;"
```

Comment:
As mentioned on the chat, you can't have `${input.conf}` == `${output.conf}`, no build system will support that.